src/platforms/common/blackpill-f4/blackpill-f4.h

```diff
@@ -27,7 +27,9 @@
 #include "timing.h"
 #include "timing_stm32.h"
 
-#define PLATFORM_HAS_TRACESWO
+#define PLATFORM_HAS_TRACESWO 1
```

Comment:
This macro is only ever used with a `#ifdef`/`#if defined()` check, never for its absolute definition value. Just defining it without value is perfectly reasonable enough.